portfolio/src/main/webapp/script.js

```diff
@@ -1,166 +1,229 @@
+google.charts.load('current', {
+  'packages': ['geochart'],
+  'mapsApiKey': 'AIzaSyC35R1Q1qkQndHm-Ni6isXLbsSCQif3Umg'
+});
+// google.charts.setOnLoadCallback(drawMarkersMap);
+google.charts.setOnLoadCallback(getChart);
+
+
+function getChart() {
+  const chartType = document.getElementById("chart-type").value;
+  if (chartType === "GeoMap") {
+    drawMarkersMap();
+  } else if (chartType === "LineChart") {
+    drawChart();
+  }
+
+}
```

Comment:
nit: Remove the empty line before the bracket and add an empty line after the bracket to separate the functions